intellij-ide.md

```diff
@@ -136,8 +136,33 @@ To edit your code and hot reload changes:
 
 1. Make desired changes in your source code (without stopping the debugger).
 
-1. In the Debug panel at the bottom, click the **Hot Reload button** (shown with
- an icon containing a lightning bolt, and a play symbol).
+1. In the Debug panel at the bottom, click the **Hot Reload button**.
+
+![Main IntelliJ toolbar](/images/intellij/debug-toolbar.png)
+
+### IntelliJ Key Mappings for _Hot Reload_ and _Full Application Restart_
```

Comment:
This heading looks very long in the table of contents compared to the others. I suggest we change it to just 'IntelliJ keyboard shortcuts'